modules/s3-logfile/variables.tf

```diff
@@ -105,6 +105,15 @@ variable "sample_rate" {
   description = "Sample rate. See https://honeycomb.io/docs/guides/sampling/."
 }
 
+variable "sample_rate_rules" {
```

Comment:
Do you think we could update the [s3-logfile test](https://github.com/honeycombio/terraform-aws-integrations/blob/main/tests/s3-logfile.tf) to use this variable for one of the two LBs we're already using?

Open to also a third LB being created if that feels nicer, but I'd like to get this new feature into the testsuite

Reply:
We aren't running a third-party tool like InSpec or terratest to validate them today. That level of effort is beyond our current testing.